dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -30,10 +30,26 @@ const ampData = (section: string, contentType: string): string => {
 	return `/${dfpAccountId}/${dfpAdUnitRoot}/amp`;
 };
 
-const getPlacementId = (adRegion: AdRegion): number => {
+/**
+ * Determine the Placement ID that is used to look up a given stored bid request
+ *
+ * Stored bid requests are stored by the prebid server instance and each is
+ * keyed by a placement ID. This placement ID corresponds to the tag id parameter
+ * provided on the client
+ *
+ * @param isSticky Whether the ad is sticky - sticky ads have stored bid requests
+ * containing different SSP ids to non-sticky ads
+ * @param adRegion The advertising region - different regions are covered by different
+ * stored bid requests
+ * @returns The placement id for an ad, depending on its ad region and whether
+ * it is sticky
+ */
+const getPlacementId = (isSticky: boolean, adRegion: AdRegion): number => {
 	switch (adRegion) {
-		case 'US':
-			return 7;
+		case 'US': {
+			// In the US use different placement IDs depending on whether ad is sticky
+			return isSticky ? 22138171 : 7;
```

Comment:
👆 note we know longer have a mapping directly from ad region to placement id, we now also take into account whether an ad is sticky.